### ISSUE TITLE
Equivalent of Fold-Change analysis?

### ISSUE THREAD

**Issue by joannawolthuis** [const-ae/proDA#16]
Hi there! 

Thanks for your hard work on this package!
I'm trying it out on my metabolomics data currently with a high amount of missing values. 
I was wondering if there was a way to make an equivalent of a volcano plot using your Wald test alongside another (not currently implemented?) test that tests something similar to fold change between conditions :) 

Kind regards and thank you for your time,
Joanna

**Comment by const-ae** [comment]
Hi Joanna,

just to make sure that I understand what you are asking: do you want to perform some alternative test and use the resulting p-value on the y-axis of a volcano plot?

**Comment by joannawolthuis** [comment]
To add: I'm asking here rather than using the completed table because I saw your comment elsewhere on that that might show an unrealistic amount of significant features :)

**Comment by fedeserral** [comment]
hi, i have the same question.

Thaks!

**Comment by const-ae** [comment]
Sorry, I am still a bit confused about this question. Is this about performing a statistical test where you check that the log-fold change is larger than some value? Similar to the  `lfcThreshold` argument in `DESeq2` (https://www.bioconductor.org/packages/release/bioc/vignettes/DESeq2/inst/doc/DESeq2.html#tests-of-log2-fold-change-above-or-below-a-threshold)?

**Comment by fedeserral** [comment]
yes, but is any way to do it using proDA? I didn't find the way.

**Comment by const-ae** [comment]
No, I haven't implemented such an argument. One would need to modify the [`run_wald_parameter_test`](https://github.com/const-ae/proDA/blob/devel/R/test_diff.R#L245) to do something like the [`DESeq2::results`](https://github.com/thelovelab/DESeq2/blob/devel/R/results.R#L493) , but I currently don't have time to implement such a feature. But I if you are interested in making a PR, I am happy to help :)

**Comment by fedeserral** [comment]
thaks for the answer! It would be great this implementation to work with proteomic data.
